sweetest/src/main/java/com/mysugr/sweetest/framework/configuration/ModuleTestingConfiguration.kt

```diff
@@ -31,6 +32,10 @@ fun moduleTestingConfiguration(
     val scope = Dsl.MainScope()
     run?.invoke(scope)
 
+    if (scope.dependencies.isEmpty()) {
```

Comment:
is it ok to remove this? i think the deprecation of the constructor taking the module testing configuration should be enough? 🤔

Reply:
Fine for me